apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -542,3 +542,17 @@ role(Pubkey, #{pub               := MyKey
         {_OtherKey, initiator} -> responder;
         {_OtherKey, responder} -> initiator
     end.
+
+wait_for_signed_transaction_in_block(Node, SignedTx) ->
+    TxHash = aetx_sign:hash(SignedTx),
+    MineTx =
+        fun Try(0) -> did_not_mine;
+            Try(Attempts) ->
+                aecore_suite_utils:mine_key_blocks(aecore_suite_utils:node_name(Node), 2),
+                case rpc(Node, aec_chain, find_tx_location, [TxHash]) of
```

Comment:
NOTE: Behavior changed of 'aec_chain:find_tx_location' in new tx GC PR.

Reply:
Yes, there is now both 'none' and 'not_found' both should crash the test...